lib/convection/model/template/resource_property/aws_ec2_iam_instance_profile.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../resource_property'
```

Comment:
Nope, this looks right. It will load the original `Convection::Model::Template::ResourceProperty` before appending to its namespace.